azure_databricks_api/__sp_auth.py

```diff
@@ -0,0 +1,35 @@
+import requests
+
+
+class ServicePrincipalAuth(object):
+
+    def __init__(self, **kwargs):
+        self._tenant_id = kwargs.pop('tenant_id')
+        self._client_id = kwargs.pop('client_id')
+        self._client_secret = kwargs.pop('client_secret')
+        self._databricks_resource_id = kwargs.pop('databricks_resource_id')
+
+    def authenticate(self):
+        auth_headers = {'Content-Type': "application/x-www-form-urlencoded"}
+        application_id = '2ff814a6-3304-4ab8-85cb-cd0e6f879c1d'
```

Comment:
Could you promote this to a class attribute and include a comment (with link) on where the application_id came from?